source/content/traffic-limits.md

```diff
@@ -28,6 +28,18 @@ For sites on Performance plans, we will notify you if your site exceeds the limi
 ### How will Pantheon let me know my site will automatically be bumped to the next level?
 You will receive a communication that you are over on month one, and again on month two when the plan level is changed.
 
+### What if my site's metrics exceed the limit of the Performance Extra Large Plan?
+The Performance Extra Large Plan allows for 300,000 monthly visits and 1.5 million monthly page views. High traffic sites should be moved to an Elite plan. To learn about moving to an Elite Plan, please [contact us](https://pantheon.io/contact-us).
+
+If you need time or are unable to commit to an annual contract, sites that exceed the Performance Extra Large limits will be upgraded to a Performance 2X Large Plan, which has a limit of 600,000 monthly visits and 3 million monthly page views. The 2X Large plan is not available for purchase via our dashboard, but can be applied by our Support team. Prices for the 2X Large plan are as follows:
+
+| Payment Type      |  Price   |
```

Comment:
Point of concern: We list monthly and annual prices here in a single column, which may be confusing. Suggest we provide the monthly cost of the annual pricing, and adjust the header to **Price Per Month**.

Reply:
Agree to this change - pls proceed.

Reply:
Adjusted in https://github.com/pantheon-systems/documentation/pull/5270/commits/3fb72443d6d227b838e510842730c267a2c3fb95